### DISCUSSION TITLE
Styled Components and svg background images

### DISCUSSION THREAD

**Discussion by gutyerrez** [vercel/next.js#19047] [Help] [answered]
Hello, i'm trying to use a svg image in background of an styled component but i this error is returned:

`
TypeError: C:\Users\Gutyerrez\Documents\Hyren Network\w.Web\src\components\icons\ArrowRight\index.ts: pass.get(...) is not a 
function
`

**Comment by jamesmosier** [comment, marked as answer]
Hi @CutyeORenegado. Do you have a code sample of what you have tried?

I am able to reference images in my styled-components like this (using [next-images](https://www.npmjs.com/package/next-images)):

```js
import LOGO from './images/my-logo.svg';

const MyElement = styled.div`
  background-image: url('${LOGO}');
`
```

or if you put the file in the `public` folder:

```js
const MyElement = styled.div`
  background-image: url('/my-logo.svg');
`
```
